e2e/static-site/static_site_suite_test.go

```diff
@@ -0,0 +1,38 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package static_site_test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/aws/copilot-cli/e2e/internal/client"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var cli *client.CLI
+var appName string
+
+const domainName = "static-site.copilot-e2e-tests.ecs.aws.dev"
```

Comment:
I assume we alraedy have this domain delegated to the static site account 👀 ?

Reply:
yeah that's right